### PR TITLE
CIAC-6419 - Fix

### PR DESCRIPTION
**Pull request by AradCarmi**
## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-6419)

## Description
Fixed an issue where adding the `ready-for-instance-test` label in the external PR the build failed if the contributor repo name is not `content`.

### CODE REVIEW

**Comment by AradCarmi** [comment]
FYI @dantavori,
Force merge is needed as we deleted the temp files causing the validate to fail on BA115.

File: .github/workflows/trigger-contribution-build.yml [hunk 1]
```diff
@@ -24,9 +24,10 @@ jobs:
...
           BASE_BRANCH: ${{ github.event.pull_request.base.ref }}
           CONTRIB_BRANCH: ${{ github.event.pull_request.head.label }}
+          CONTRIB_REPO: ${{ github.event.repository.name }}
           USERNAME: ${{ secrets.SECRET_CHECK_USER_NG }}
           PASSWORD: ${{ secrets.SECRET_CHECK_PASS_NG }}
           GOLD_SERVER_URL: ${{ secrets.GOLD_SERVER_URL_NG }}
         run: |
           echo "Trigger contribution build for PR: $PR_NUMBER with base branch: $BASE_BRANCH contrib branch: $CONTRIB_BRANCH"
-          poetry run python ./Utils/github_workflow_scripts/trigger_contribution_build.py --pr_number $PR_NUMBER --base_branch $BASE_BRANCH --contrib_branch $CONTRIB_BRANCH --username $USERNAME --password $PASSWORD --gold_server_url $GOLD_SERVER_URL
+          poetry run python ./Utils/github_workflow_scripts/trigger_contribution_build.py --pr_number $PR_NUMBER --base_branch $BASE_BRANCH --contrib_branch $CONTRIB_BRANCH --contrib_repo $CONTRIB_REPO --username $USERNAME --password $PASSWORD --gold_server_url $GOLD_SERVER_URL
```
Comment (dantavori) [inline .github/workflows/trigger-contribution-build.yml:33]: does it trigger in both gitlab and circle?

File: Utils/github_workflow_scripts/trigger_contribution_build.py [hunk 2]
```diff
@@ -40,7 +42,7 @@ def trigger_generic_webhook(options):
     body = {
         "name": "GenericWebhook_trigger_contribution_build",
         "raw_json": {"BaseBranch": base_branch, "PullRequestNumber": pr_number, "ContribBranch": contrib_branch,
-                     "ProjectID": GITLAB_PROJECT_ID},
+                     "ProjectID": GITLAB_PROJECT_ID, "ContribRepo": contrib_repo},
```
Comment (dantavori) [inline Utils/github_workflow_scripts/trigger_contribution_build.py:45]: if I understand correctly, this will allow defining the env variable in both builds, right? does it happen in gold? a bit hard to follow

**Review by dantavori — APPROVED** [review]
looks good, let's sync on my questions before you merge
@RotemAmit - worth having a glance as well

File: Utils/github_workflow_scripts/trigger_contribution_build.py (hunk 2 above)
Comment (AradCarmi) [reply Utils/github_workflow_scripts/trigger_contribution_build.py:45]: This request is for our Gold (`GenericWebHook`) and it will be mapped (using the `Contribution Build Incoming Mapper`) to a new incident field (`incident_contributionreponame`) and will eventually get to our Gitlab pipeline.

File: .github/workflows/trigger-contribution-build.yml (hunk 1 above)
Comment (AradCarmi) [reply .github/workflows/trigger-contribution-build.yml:33]: WDYM? 
This triggers the Gold `GenericWebHook`.
